packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -965,9 +965,29 @@ export class Users {
       _.values(_.mapValues(profitLossResultsByMarketAndOutcome, _.values))
     );
 
+    const marketFinalizedResults = await db.MarketFinalized.where('market')
+      .anyOf(_.map(allProfitLossResults, 'market'))
+      .toArray();
+    const marketFinalizedByMarket = _.keyBy(marketFinalizedResults, 'market');
+    const marketsData = await db.Markets.where('market')
+      .anyOf(_.map(allProfitLossResults, 'market'))
```

Comment:
this could just be the smaller set of markets from `marketFinalizedResults` insteadI think

Reply:
` .anyOf(_.map(allProfitLossResults, 'market'))` filter finalized market by profit and loss markets.